subworkflows/DATA_TO_MODEL.nf

```diff
@@ -3,11 +3,13 @@
 nextflow.enable.dsl = 2
 
 // Synapse ID for Submission View
-params.view_id = "syn52576179"
+params.view_id = "syn52658661"
 // Scoring Script
-params.scoring_script = "score.py"
+params.scoring_script = "data_to_model_score.py"
```

Comment:
So for the dynamic challenge, the parameters would be `dynamic_challenge_score.py`?